cobalt/build/test_targets.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+# Copyright 2024 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Calculates test targets based on a root package from GN dependencies."""
+
+import argparse
+import json
+import os
+import typing
+
+import networkx as nx
+
+_UNRELATED_TARGETS_PACKAGES = {'components', 'third_party', 'ui'}
```

Comment:
What effect does this have ? E.g. what happens if things get added here ?

Maybe leave a comment

Also _technically_ that maybe should be a set, not a dictionary